src/main/site/assets/less/layout.less

```diff
@@ -220,6 +220,7 @@ ul {
 
   & > li {
     list-style-type: none;
+    margin-top: 0.8em;
```

Comment:
As this is impacting both `ul` and `ol` with the same style, you should move it inside the common declaration (after line 215). Same goes with the `& > p:first-child`